src/configure/configure.ts

```diff
@@ -110,7 +112,7 @@ class Orchestrator {
             await pipelineConfigurer.createPreRequisites(this.inputs, !!this.azureResourceClient ? this.azureResourceClient : new AppServiceClient(this.inputs.azureSession.credentials, this.inputs.azureSession.environment, this.inputs.azureSession.tenantId, this.inputs.subscriptionId));
 
             telemetryHelper.setCurrentStep('CreateAssets');
-            if (this.inputs.pipelineConfiguration.template.templateType === TemplateType.REMOTE) {
+            if (this.inputs.pipelineConfiguration.template.templateType === TemplateType.REMOTE && this.inputs.sourceRepository.repositoryProvider === RepositoryProvider.Github) {
```

Comment:
please explain the fix. The bug says that this happens when repo provider is github. But with the change, the flow will still go in this block.

Reply:
The bug says -> Repo provider is ADO and repository is GitHub then this issue is happening.

Reply:
What if the repoProvider is GitHub but ``` extensionVariables.enableGitHubWorkflow ``` is false
Is this fix handling thise scenario? If not, then the condition can be on the type of pipelineConfigurer insread of repo provider.

Reply:
@kanika1894 
if extensionVariables.enableGitHubWorkflow is false, then how can repositoryProvider is GitHub?
I think you are confusing repositoryProvider with repo type.